src/App.jsx

```diff
@@ -1,24 +1,61 @@
-import {useEffect, useState} from 'react'
+import {useEffect, useRef, useState} from 'react'
 import { SignClient } from '@walletconnect/sign-client';
 import { Web3Modal } from '@web3modal/standalone';
 import './App.css'
 
 
 // Your dapp's Project ID from https://cloud.walletconnect.com/
-const WC_PROJECT_ID = '9e1152b9dc0318eea105dc31238fbc00';
+const WC_PROJECT_ID = 'cc3b2ecffec2ba45dfa80295a62b3f5a';
 const CHAIN_ID = 'finschia-2';
+
 // https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md
 const CAIP_BLOCKCHAIN_ID = `cosmos:${CHAIN_ID}`
-const signClient = await SignClient.init({
+
+const web3modal = new Web3Modal({
+    walletConnectVersion: 2,
     projectId: WC_PROJECT_ID,
-    metadata: {
```

Comment:
@tvd12 do you need to remove this metadata?

Reply:
because it's here: https://github.com/dosivault/wc_v2_example/pull/1/files#diff-d274a54187c91ba0f532df2a9e194e27ab50e988f5e4c33f5a7893918320c661R30